cost-analyzer/values.yaml

```diff
@@ -83,6 +83,7 @@ kubecostModel:
   imagePullPolicy: Always
   warmCache: true
   warmSavingsCache: true
+  etl: true
```

Comment:
what about etlEnabled? I'm supportive either way, but just wanted to throw out the suggestion.

Reply:
I'm open to other names, but I think we should be consistent with "Enabled" as a suffix and add that to all of the enable/disable flags, or else have it on none of them.

Reply:
Your call... happy to keep as is